streams/voxel_stream_region_files.cpp

```diff
@@ -508,6 +508,7 @@ static void migrate_region_meta_data(Dictionary &data) {
 VoxelFileResult VoxelStreamRegionFiles::load_meta() {
 
 	ERR_FAIL_COND_V(_directory_path == "", VOXEL_FILE_CANT_OPEN);
+	if (_directory_path.empty()) return VOXEL_FILE_CANT_OPEN;
```

Comment:
I'm pretty sure this if statement is never going to run, as `ERR_FAIL_COND_V` already returns in the case that path is empty.